doc/api/util.md

```diff
@@ -2008,11 +2013,20 @@ changes:
     the built-in behavior, from adding additional checks through to reprocessing
     the tokens in different ways.
     **Default:** `false`.
+  * `help` {string} General help text to display at the beginning of help output.
+  * `addHelpOption` {boolean} Whether to automatically add a help option to the
+    options if general help text is provided and no existing help option is defined.
+    The auto-added help option uses `-h` short flag and `--help` long flag.
+    **Default:** `true` if `help` is provided, `false` otherwise.
+  * `returnHelpText` {boolean} Whether to return formatted help text in the result.
+    **Default:** `true` if `help` is provided or `addHelpOption` is `true`, `false` otherwise.
```

Comment:
what would be the cost for the case where you provide help text, and simply ignore the returned help text?